specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-11-01-preview3/communicationserviceschat.json

```diff
@@ -0,0 +1,1589 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Chat Service",
+    "description": "Azure Communication Chat Service",
+    "contact": {
+      "email": "acsdevexdisc@microsoft.com"
+    },
+    "version": "2020-11-01-preview3"
+  },
+  "paths": {
+    "/chat/threads/{chatThreadId}/readreceipts": {
+      "get": {
+        "tags": [
+          "Threads"
+        ],
+        "summary": "Gets chat message read receipts for a thread.",
+        "operationId": "ListChatReadReceipts",
```

Comment:
Let's switch back to Noun_Verb format and that allow us to group the methods and split the generated code to align with the convenience layer. Eg: "Chat_CreateThread" and "ChatThread_SendMessage" would make the generator to create to clients, one for Chat and one for ChatThread

Reply:
This is fixed now.